### ISSUE TITLE
CLI option for repository?

### ISSUE THREAD

**Issue by khullah** [FredrikNoren/ungit#1006]
And I open yet another issue.. don't worry, its cause I love this project ;)

When running command `ungit` in shell, I want to set the repository on the go. `ungit "path"` doesn't work and shows a list of parameters. Parameters `--urlBase` and `--rootPath` do other things, and I didn't find one for what I want specifically. Is there such an option?

**Comment by jung-kim** [comment]
If you have `launchBrowser` to true and `cd` to the git repo then `ungit` command will launch the browser with the git repo. 

But would you rather have direct launch command?

(Also, there are some config propagation problems that I would need to fix so if you have `.ungitrc` file command line args may not work.)

**Comment by khullah** [comment]
Well, yes, going to the path with `cd` is the expected way of doing it, but
what if I'm not in the shell? Eg directly through "run command" or even
inside a link/alias? Also, even if in a batch file, it would be more
elegant the one liner than possibly several cd and drive change, etc.

I assume `launchBrowser` is default? About `.ungitrc`, I guess you would
have to load that as well, yes.

**Comment by jung-kim** [comment]
Once #1007 is merged, you can pass in `ungit --forceLaunchPath=${path}` to launch ungit at any path you would like it to open with.

And yes default value of `launchBrowser` is true [link](https://github.com/FredrikNoren/ungit/blob/master/source/config.js#L53).
